corpus/expressions.txt

```diff
@@ -378,7 +378,7 @@ yield from a
       (expression_list (identifier))
       (yield (expression_list (integer)))))
   (expression_statement
-    (yield (identifier))))
+    (yield (expression_statement (identifier)))))
```

Comment:
@maxbrunsfeld Thanks for the review! So for nesting of `yield`, I think you have to use the Python3 `yield from` syntax. This is syntactically correct in Python3:

```python
def example():
    yield from (yield from (x for x in range(1, 10)))
```

The trouble was since `yield from ...` can contain what we currently define as `expression_statement`, then matching against `_expression` is not enough (the original parse error was related to a `yield from` statement containing a nested `expression_statement`).

I think this way of structuring the grammar hits the sweet spot of being simple while preserving correctness, but I agree, the example `(yield (expression_statement (identifier)))` is a bit strange. The problem lies in that we currently model all `_expression` as `expression_statement`. 

I'll play with removing `_expression` from `expression_statement`, and instead choice on `_expression` directly in rules contained by `expression_statement`. That could prevent us from wrapping single identifiers with `expression_statement` in places that would be unintuitive.